common/src/main/java/com/netflix/conductor/common/config/ObjectMapperConfiguration.java

```diff
@@ -34,6 +33,5 @@ public void customizeDefaultObjectMapper() {
         objectMapper.setDefaultPropertyInclusion(
                 JsonInclude.Value.construct(
                         JsonInclude.Include.NON_NULL, JsonInclude.Include.ALWAYS));
-        objectMapper.registerModule(new AfterburnerModule());
```

Comment:
Why is this module removed?

Reply:
Found issues with JDK17 + SB3 and other dependencies.